### PR TITLE
chore(location): standardize abbreviated parameter

### PR DESCRIPTION
**Pull request by Shinigami92**
related to #1349

### CODE REVIEW

**Comment by ST-DDT** [comment]
There are lint errors.

**Review by ST-DDT — DISMISSED** [review]

File: src/modules/location/index.ts [hunk 1]
```diff
@@ -676,15 +676,77 @@ export class LocationModule {
...
+   *
+   * @since 8.0.0
+   *
+   * @deprecated Use `faker.location.direction({ abbreviated })` instead.
+   */
+  direction(abbreviated?: boolean): string;
+  /**
+   * Returns a random direction (cardinal and ordinal; northwest, east, etc).
+   *
+   * @param options Whether to use abbreviated or an options object.
```
Comment (xDivisionByZerox) [inline src/modules/location/index.ts]:
```suggestion
   * @param options Whether to use abbreviated or an options object. Defaults to `{}`.
```

File: src/modules/location/index.ts [hunk 2]
```diff
@@ -676,15 +676,77 @@ export class LocationModule {
...
+           *
+           * @default false
+           */
+          abbreviated?: boolean;
+        }
+  ): string;
+  /**
+   * Returns a random direction (cardinal and ordinal; northwest, east, etc).
+   *
+   * @param options Whether to use abbreviated or an options object.
```
Comment (xDivisionByZerox) [inline src/modules/location/index.ts]:
```suggestion
   * @param options Whether to use abbreviated or an options object. Defaults to `{}`.
```

File: src/modules/location/index.ts [hunk 3]
```diff
@@ -721,15 +789,77 @@ export class LocationModule {
...
+   *
+   * @since 8.0.0
+   *
+   * @deprecated Use `faker.location.cardinalDirection({ abbreviated })` instead.
+   */
+  cardinalDirection(abbreviated?: boolean): string;
+  /**
+   * Returns a random cardinal direction (north, east, south, west).
+   *
+   * @param options Whether to use abbreviated or an options object.
```
Comment (xDivisionByZerox) [inline src/modules/location/index.ts]:
```suggestion
   * @param options Whether to use abbreviated or an options object. Defaults to`{}`. 
```

File: src/modules/location/index.ts [hunk 4]
```diff
@@ -721,15 +789,77 @@ export class LocationModule {
...
+           *
+           * @default false
+           */
+          abbreviated?: boolean;
+        }
+  ): string;
+  /**
+   * Returns a random cardinal direction (north, east, south, west).
+   *
+   * @param options Whether to use abbreviated or an options object.
```
Comment (xDivisionByZerox) [inline src/modules/location/index.ts]:
```suggestion
   * @param options Whether to use abbreviated or an options object. Defaults to `{}`.
```

**Review by xDivisionByZerox — APPROVED** [review]

**Comment by xDivisionByZerox** [comment]
What about:
- date.month({ abbr })
- date.weekday({ abbr })

**Review by ST-DDT — APPROVED** [review]

**Review by matthewmayer — APPROVED** [review]

**Comment by ST-DDT** [comment]
> What about:
> 
>     * date.month({ abbr })
> 
>     * date.weekday({ abbr })

We should fix them as well, but in a different PR.

**Pull request merged**